.github/workflows/build_wheels.yml

```diff
@@ -15,38 +15,52 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macOS-10.15, ubuntu-20.04, windows-2019]
-
+        os: [ubuntu-latest, macos-latest, windows-latest]
     steps:
       - uses: actions/checkout@v2
 
-      # Used to host cibuildwheel
       - uses: actions/setup-python@v2
 
       - name: Install cibuildwheel
         run: python -m pip install cibuildwheel==1.9.0
 
+      - name: Clone pybind11 repo
+        run: git clone https://github.com/pybind/pybind11.git
+
       - name: Install sundials on macOS
         if: matrix.os == 'macOS-10.15'
         run: |
           brew update
           brew install sundials
```

Comment:
Perhaps later all OSs should get sundials from the vcpkg registry.  Homebrew is very trigger happy on updating to potentially incompatible versions

Reply:
Yes. Building the `idaklu` extension (which depends on sundials) is the only thing that made me have a look at vcpkg. But now that it is setup, I can't think of a reason why this shouldn't be used on other platforms as well. This would really simplify the process.. currently the sundials build is different on the the 3 OSs!